src/pages/Home.jsx

```diff
@@ -38,7 +38,7 @@ export default function Home() {
             width: "50%",
           }}
         >
-          <h1>Spring 2023 Applications Now Open!</h1>
+          <h1>Fall 2023 Applications Open August 23!</h1>
 
           <a
```

Comment:
I think we should either add the new link, or comment out the old link so that people can't still access the old application.